hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/clients/ClientRegistry.java

```diff
@@ -17,7 +17,7 @@
 
   Channel getConfigServiceChannel();
 
-  TraceEntityReader<StructuredTrace, Event> getEntityReader();
+  TraceEntityAccessor getTraceEntityAccessor();
```

Comment:
Since this is both reading and writing, tried my hand at a rename... happy to take suggestions though!